symengine/lib/symengine_wrapper.pyx

```diff
@@ -833,6 +864,162 @@ cdef class Constant(Basic):
             raise Exception("Unknown Constant")
 
 
+class Boolean(Basic):
+    pass
+
+
+class BooleanTrue(Boolean):
+
+    def _sympy_(self):
+        import sympy
+        return sympy.S.true
+
+    def _sage_(self):
+        return True
+
+
+class BooleanFalse(Boolean):
+
+    def _sympy_(self):
+        import sympy
+        return sympy.S.false
+
+    def _sage_(self):
+        return False
+
+
+class Relational(Boolean):
+    pass
+
+Rel = Relational
+
+
+class Equality(Relational):
+
+    def __new__(cls, *args):
+        return eq(*args)
+
+    def _sympy_(self):
+        import sympy
+        s = self.args_as_sympy()
+        return sympy.Equality(*s)
+
+    def _sage_(self):
+        import sage.all as sage
+        s = self.args_as_sage()
+        return sage.eq(*s)
+
+    func = __class__
+
+
+Eq = Equality
+
+
+class Unequality(Relational):
+
+    def __new__(cls, *args):
+        return ne(*args)
+
+    def _sympy_(self):
+        import sympy
+        s = self.args_as_sympy()
+        return sympy.Unequality(*s)
+
+    def _sage_(self):
+        import sage.all as sage
+        s = self.args_as_sage()
+        return sage.ne(*s)
+
+    func = __class__
+
+
+Ne = Unequality
+
+
+class GreaterThan(Relational):
```

Comment:
Then make it a function, otherwise `isinstance(GreaterThan(x, y), GreaterThan)` will return false instead of throwing an error